### ISSUE TITLE
Consider refactoring plot-scraping functionality into other ContentMine components

### ISSUE THREAD

**Issue by ghost** [ContentMine/svg2xml#4]
@petermr notes that some or all of this functionality might be better placed elsewhere in the ContentMine stack.

**Comment by tarrow** [comment]
Do you have a proposal as to where is should go?

I guess further down? Norma?

**Comment by ghost** [comment]
@tarrow no, sorry, no ideas yet. @petermr just flagged it up to me today as something ContentMine coders may want to look at in the future, when more pressing tasks have been taken care of :)
